editor-service/src/main/webapp/WEB-INF/views/include/scripts.jsp

```diff
@@ -1,12 +1,12 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
 
-    <script src="<c:url value='/resources/js/libs/jquery/jquery-1.11.3.min.js' />"></script>
-    <script src="<c:url value='/resources/js/libs/joint/joint.js' />"></script>
-    <script src="<c:url value='/resources/js/libs/jquery/line/jquery.line.js' />"></script>
-    <script src="<c:url value='/resources/js/libs/jquery/treeview/jquery.treeview.js' />"></script>
-    <script src="<c:url value='/resources/js/libs/jquery/ui/jquery-ui.min.js' />"></script>
+    <%--<script src="<c:url value='/resources/js/libs/jquery/jquery-1.11.3.min.js' />"></script>--%>
+    <%--<script src="<c:url value='/resources/js/libs/joint/joint.js' />"></script>--%>
+    <%--<script src="<c:url value='/resources/js/libs/jquery/line/jquery.line.js' />"></script>--%>
+    <%--<script src="<c:url value='/resources/js/libs/jquery/treeview/jquery.treeview.js' />"></script>--%>
+    <%--<script src="<c:url value='/resources/js/libs/jquery/ui/jquery-ui.min.js' />"></script>--%>
     <script src="<c:url value='/resources/js/libs/context-menu/context-menu.js' />"></script>
-    <script src="<c:url value='/resources/js/libs/bootstrap/bootstrap.min.js' />"></script>
+    <%--<script src="<c:url value='/resources/js/libs/bootstrap/bootstrap.min.js' />"></script>--%>
     <script src="<c:url value='/resources/js/libs/raphael/raphael-min.js' />"></script>
```

Comment:
better move all those libs to Require.js (if possible, if not -- use hack used for joint)